scss/composer/_composer.scss

```diff
@@ -1,12 +1,10 @@
 .composer {
-  background-color: var(--composerBg);
```

Comment:
let's keep this line, whatever we do.

it is much clearer when "quote" + "image or other attachment" + "text and send button" is recognoised as one things.

this is also better for inner consistency (and for quotes also for external consistency)